kube/services/jupyterhub/jupyterhub_config.py

```diff
@@ -107,3 +136,15 @@
 c.JupyterHub.hub_ip = "0.0.0.0"
 c.RemoteUserAuthenticator.auth_refresh_age = 1
 c.RemoteUserAuthenticator.refresh_pre_spawn = True
+
+c.KubeSpawner.env_keep = ['HOSTNAME', 'POD_NAMESPACE']
+
+c.KubeSpawner.extra_containers = [{
```

Comment:
add a `fuseEnabled` flag to the `jupyter` block of `manifest.json` (you can pass that into the jupyterhub container as an environment variable by modifying the jupyterhub-deploy.yaml), and only run the block that adds the fuse-sidecar if `fuseEnabled` is set true - ex:
```
if os.environ.get('fuseEnabled') == 'yes':
    c.KubeSpawner.extra_containers = [ ... ]
```

Reply:
I don't think we have jupyter that doesn't need fuse sidecar. So I don't see a flag is needed..